Source/WebGPU/WebGPU/Queue.swift

```diff
@@ -28,11 +28,11 @@ private let largeBufferSize = 32 * 1024 * 1024
 public func writeBuffer(
     queue: WebGPU.Queue, buffer: WebGPU.Buffer, bufferOffset: UInt64, data: SpanUInt8
 ) {
-    queue.writeBuffer(mtlBuffer: buffer.buffer(), bufferOffset: bufferOffset, data: data)
+    queue.writeBuffer(buffer: buffer, bufferOffset: bufferOffset, data: data)
 }
 
 extension WebGPU.Queue {
-    public func writeBuffer(mtlBuffer: any MTLBuffer, bufferOffset: UInt64, data: SpanUInt8) {
+    public func writeBuffer(buffer: WebGPU.Buffer, bufferOffset: UInt64, data: SpanUInt8) {
```

Comment:
```suggestion
    public func writeBuffer(_ buffer: WebGPU.Buffer, bufferOffset: UInt64, data: SpanUInt8) {
```

I think the `buffer` parameter name can be omitted since it's in the function name